docs/_documentations/performance.md

```diff
@@ -20,13 +20,13 @@ You will learn how to add and enable monitoring to your application. Then, you c
 
 ## Prerequisites
 
-- VS Code or Eclipse
+- VS Code, Eclipse, or Eclipse Che
```

Comment:
Does it work in IntelliJ? (Is there a menu option there to open the dashboard webpage?)

Reply:
IntelliJ -- not yet

Reply:
Stet